xml/System.Net.Http/HttpClient.xml

```diff
@@ -67,7 +67,29 @@ public class GoodController : ApiController
   
 ```  
   
-   
+The <xref:System.Net.Http.HttpClient> is a high-level API that wraps the lower-level functionality available on each platform where it runs.
+
+On each platform, <xref:System.Net.Http.HttpClient> tries to use the best available transport:
+
+| **Host/Runtime**  | **Backend**                                                                             |
+| ----------------- | --------------------------------------------------------------------------------------- |
+| Windows/.NET      | <xref:System.Net.HttpWebRequest>                                                        |
+| Windows/Mono      | <xref:System.Net.HttpWebRequest>                                                        |
+| Windows/UWP       | Windows native <xref:Syste.Net.Http.WinHttpHandler (HTTP 2.0 capable)                   |
+| Windows/.NET Core | Windows native <xref:Syste.Net.Http.WinHttpHandler (HTTP 2.0 capable)                   |
+| Android/Xamarin   | Build time selection;
+Can use either <xref:System.Net.HttpWebRequest> or be configured to use Android's native [`HttpURLConnection`](https://developer.xamarin.com/api/type/Java.Net.HttpURLConnection/) |
+| iOS, tvOS/Xamarin | Build time selection;
+Can use either <xref:System.Net.HttpWebRequest> or be configured to use Apple's [`NSUrlSession`](https://developer.xamarin.com/api/type/MonoTouch.Foundation.NSUrlSession/) (HTTP 2.0 capable) |
+| macOS/Xamarin     | Build time selection;
+Can use either <xref:System.Net.HttpWebRequest> or be configured to use Apple's [`NSUrlSession`](https://developer.xamarin.com/api/type/MonoTouch.Foundation.NSUrlSession/) (HTTP 2.0 capable) |
+| macOS/Mono        | `HttpWebRequest`                                                                        |
+| macOS/.NET Core   | `libcurl`-based HTTP transport (HTTP 2.0 capable)                                       |
+| Linux/Mono        | <xref:System.Net.HttpWebRequest>                                                        |
+| Linux/.NET Core   | `libcurl`-based HTTP transport (HTTP 2.0 capable)                                       |
+
+Users can also configure a specific transport for <xref:System.Net.Http.HttpClient> by invoking the constructor that takes a 
```

Comment:
a HttpMessageHandler -> an HttpMessageHandler

you could link to the constructors page like this `<xref:System.Net.Http.HttpClient.%23ctor*>` if you want